.azure/scripts/build.sh

```diff
@@ -57,6 +57,8 @@ if [ -n "$CHANGED_DERIVED" ] || [ -n "$GENERATED_FILES" ] ; then
     exit 1
 fi
 
+source .azure/scripts/publish_check.sh
```

Comment:
I _think_  I should move this to before the mvn step.

Reply:
Yeah, that way it should fail faster